src/routes/+page.svelte

```diff
@@ -1,2 +1,46 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<div class="relative h-hero">
+	<div class="background" />
+	<div class="title-container">
+		<div class="title-content">
+			<h1>Misogyny_DB</h1>
+			<p class="subtitle">Like IMDB but calling out misogyny in the film industry.</p>
+			<button> Add an entry </button>
```

Comment:
Try to avoid buttons outside forms. If we not already have create a component that looks like a button, but is at least a link. Otherwise you have to wrap the button in a form, to make it useable without js